bokehjs/test/models/tools/toolbar.coffee

```diff
@@ -46,6 +46,16 @@ describe "Toolbar", ->
       expect(@hover_2.active).to.be.false
       expect(@hover_3.active).to.be.false
 
+    it "should have autohide as False by default", ->
+      toolbar = new Toolbar({tools:[@hover_1, @hover_2, @hover_3]})
+      expect(toolbar.autohide).to.be.false
+      expect(toolbar.visible).to.be.true
+
+    it "should have autohide as False by default", ->
```

Comment:
is this test name intended to be identical to the previous one?

Reply:
no...